src/cache.ts

```diff
@@ -26,12 +26,12 @@ export async function setCachedFile(fileName: string, value: Buffer): Promise<vo
 }
 
 export async function getCachedFile(fileName: string, encoding?: BufferEncoding): Promise<Buffer> {
-	let cachedFile: Buffer = Buffer.alloc(0);
+	let cachedFile = Buffer.alloc(0);
```

Comment:
ok I don't know how typescript works but can you make sure this isnt a joke or something lol

Reply:
That's something called type inferencing - uses the context of the assignment to determine typing.

Reply:
What are you talking about?